File: Semester4/web/Practical Exam/JSP/sport/src/main/java/com/example/sport/model/TeamMembers.java
package com.example.sport.model;

public class TeamMembers {
    private int id;
    private int idPlayer1;
    private int getIdPlayer2;
    private int idTeam;

    public TeamMembers(int id, int idPlayer1, int getIdPlayer2, int idTeam) {
        this.id = id;
        this.idPlayer1 = idPlayer1;
        this.getIdPlayer2 = getIdPlayer2;
        this.idTeam = idTeam;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdPlayer1() {
        return idPlayer1;
    }

    public void setIdPlayer1(int idPlayer1) {
        this.idPlayer1 = idPlayer1;
    }

    public int getGetIdPlayer2() {
        return getIdPlayer2;
    }

    public void setGetIdPlayer2(int getIdPlayer2) {
        this.getIdPlayer2 = getIdPlayer2;
    }

    public int getIdTeam() {
        return idTeam;
    }

    public void setIdTeam(int idTeam) {
        this.idTeam = idTeam;
    }
}

File: Semester4/web/Practical Exam/PHP/sport/server/model/TeamMembers.php
<?php

class TeamMembers implements JsonSerializable{
    private int $id;
    private int $idPlayer1;
    private int $idPlayer2;

    private int $idTeam;



    public function __construct(int $id, int $idPlayer1, int $idPlayer2){
        $this->id = $id;
        $this->idPlayer1 = $idPlayer1;
        $this->idPlayer2 = $idPlayer2;
    }

    public function getId(): int{
        return $this->id;
    }

    public function getIdPlayer1(): int{
        return $this->idPlayer1;
    }

    public function getIdPlayer2(): int{
        return $this->idPlayer2;
    }

    public function getIdTeam(): int{
        return $this->idTeam;
    }

    public function jsonSerialize(){
        return [
            "id" => $this->id,
            "idPlayer1" => $this->idPlayer1,
            "idPlayer2" => $this->idPlayer2,
            "idTeam" => $this->idTeam,

        ];
    }

}
?>